cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2841,13 +2841,15 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload IN (OneDrive, SharePoint) Operation IN ("AnonymousLinkUsed","SharingLinkUsed") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") \
+search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload IN (OneDrive, SharePoint) Operation IN ("AnonymousLinkUsed","SharingLinkUsed") \
 |`cs_o365_link_accessed_by_external_user_internal_filter` \
-| search UserId=anonymous \
 | iplocation ClientIP \
 | fillnull value="" Country, Region, City \
 | rename ClientIP as Source \
 | stats values(Operation) as Operations, values(ObjectId) as ObjectId , values(SharingLinkScope) as SharingLinkScope, values(Permission) as Permission, values(ApplicationDisplayName) as ApplicationDisplayName, values(Source) as Source,values(EventData) as EventData, values(ItemType) as ItemType,values(EventSource) as EventSource latest(_time) as LastTime earliest(_time) as FirstTime, values(Country) as Country, values(City) as City, values(Region) as Region by SourceFileName, UserId \
+| rex field=UserId "(?<email_user>[^@]+)@(?<email_domain>.+)" \
+| eval email_domain_search = replace(email_domain, "\.", "_") | fillnull value="NOTHING_HERE_NO" email_domain_search \
```

Comment:
This logic seems it's ignoring all event which has email domain in UserId field. It is not validating which domain it is.

I think it will do similar to the following filter:
| search UserId=anonymous
| search NOT SharingLinkScope="Organization"

Reply:
| where !like(ObjectId, "%".email_domain_search."%")

This code is doing that.

Reply:
My bad, I was testing the search where mistakenly \ was replaced.